lib/fencing/st_lha.c

```diff
@@ -261,8 +268,9 @@ stonith__lha_metadata(const char *agent, int timeout, char **output)
                                    meta_shortdesc, meta_param,
                                    timeout_str, timeout_str, timeout_str);
 
-        g_free(meta_longdesc_esc);
-        g_free(meta_shortdesc_esc);
+        free(meta_longdesc);
+        free(meta_shortdesc);
```

Comment:
I just realized these are a problem ... we have to use free() or g_free() depending on how they were created